scripts/compile-clean.js

```diff
@@ -11,6 +11,7 @@
 
 const { rimraf } = require('rimraf');
 
+rimraf.sync('.cache-loader');
```

Comment:
I added this, as it is something I would expect the `yarn clean` command to do. I was confused as to why my main clone wasn't working but a clean clone was working, and this was the reason. Just makes it easier for us to start off from a clean slate.